Languages/IronPython/IronPython/Modules/_ast.cs

```diff
@@ -876,6 +887,11 @@ public abstract class @operator : AST
         [PythonType]
         public abstract class slice : AST
         {
+            public AstExpression AttachLoc(AstExpression other)
```

Comment:
There were three additional commits, I think he made his comment regarding:
First, fixing #1285 in my branch as part of my code which caused a spacing conflict.
A second and third, merging from the upstream main to mine, and from my main to this branch after the fix was introduced there.
I just removed those (the fix is already on the main branch, there are no conflicts with current state and I removed the last unintended extra 3 commits)